packages/client/hmi-client/src/components/workflow/ops/interventions/tera-intervention-card.vue

```diff
@@ -62,15 +62,15 @@
 						<div class="flex align-items-center pt-2 pb-2 gap-2">
 							<tera-input
 								type="number"
-								:model-value="i.threshold"
-								@update:model-value="(val) => onUpdateThreshold(val, index)"
+								:model-value="i.value"
+								@update:model-value="(val) => onUpdateValue(val, index)"
```

Comment:
Just noticed this, in normal circumstances we don't want the `v-for` to key on the iterating index unless the list itself is static. When you key on the index of the loop, you will get into various vue-cache issues if the list is allowed to shrink or to grow, which I think will happen in this case.

Also note what we discussed about maintainability cause you need to do this in multiple places 😄

Reply:
Is there a standard we have if we don't have a unique indicator on the data object?  All StaticInterventions just have this structure: 
```
{
    value: number
    threshold: number
}
```
Normally Im using the index as a fallback if there isn't a unique way to identify the object.  Should we store ids on the backend, or even preprocess the data with ids?

Reply:
I think you can either:
- reassign the array reference when modifications happens, eg: don't modify stuff in place, or
- create a composite key of "value:threshold"

Reply:
@mwdchang I think it should be reassigning the array reference on update and delete now with the `key fixes?` commit. 

I tried doing the composite key but since some of the keys could be the same it was screwing up the UI

Reply:
okay, so long as we have tested non-trivial insert/delete/update combinations.

Reply:
ya, Ive tested it out with a bunch of card and different intervention combos and it seems to be working fine